app/models/opensearch.rb

```diff
@@ -2,8 +2,9 @@ class Opensearch
   SIZE = 20
   MAX_PAGE = 200
 
-  def search(from, params, client, index = nil)
+  def search(from, params, client, highlight = false, index = nil)
```

Comment:
Rubocop notes that for boolean parameters we should consider keyword arguments. I suspect that's worth considering but felt like moving `index` to a keyword would make sense and while simple in the code would require a lot of test updates which I didn't want to introduce in this changeset. The suggestion is logged in code climate as technical debt and rather than _also_ opening a Jira ticket I thought we might introduce a practice in EngX of actually looking back on our code climate logged debt more frequently to look for maintenance work. Thoughts?

Reply:
Yeah, that's an interesting idea. I feel like if we were to open Jira tickets for every piece of technical debt, our backlogs would get flooded.